adapters/repos/db/batch_integration_test.go

```diff
@@ -85,6 +85,8 @@ func TestBatchPutObjectsWithDimensions(t *testing.T) {
 
 	simpleInsertObjects(t, repo, "ThingForBatching", 123)
 
+	time.Sleep(5 * time.Second)
```

Comment:
can we change `time.Sleep` into `assert.EventuallyWithT`. 
Can we do this here and in 466 line?